certbot-ci/certbot_integration_tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+"""
+General conftest for pytest execution of all integration tests lying
+in the certbot_integration tests package.
+As stated by pytest documentation, conftest module is used to set on
+for a directory a specific configuration using built-in pytest hooks.
+"""
+import os
+import json
+from _pytest import config as config_pytest
+
+
+def pytest_configure(config):
+    # type: (config_pytest.Config) -> None
+    """
+    Standard pytest hook used to add a configuration logic for each node of a pytest run.
+    :param pytest_config.Config config: the current pytest configuration
+    """
+    if not os.environ.get('CERTBOT_ACME_TYPE'):
```

Comment:
Nope it is not the same. Here we are testing the existence of `CERTBOT_ACME_TYPE`, not `CERTBOT_ACME_XDIST`. Environment variable `CERTBOT_ACME_TYPE` is not used in part 1, but it will be useful in part 3 to skip seamlessly some incompatible tests with current acme server type.